
# HG changeset patch
# User Sam Lantinga <slouken@libsdl.org>
# Date 1542069298 28800
# Node ID d274fa9731b17af90b3b95a211d327c307d90360
# Parent  5a95fbfd3617b3a309cd3f3a37143bde98872e28
Fixed bug 4367 - compatibility version decreased between 2.0.8 and 2.0.9

diff -r 5a95fbfd3617 -r d274fa9731b1 build-scripts/ltmain.sh
--- build-scripts/ltmain.sh	Sat Nov 10 20:56:23 2018 -0400
+++ build-scripts/ltmain.sh	Mon Nov 12 16:34:58 2018 -0800
@@ -7404,11 +7404,8 @@
 	  # Darwin ld doesn't like 0 for these options...
 	  func_arith $current + 1
 	  minor_current=$func_arith_result
-	  #xlcverstring="${wl}-compatibility_version ${wl}$minor_current ${wl}-current_version ${wl}$minor_current.$revision"
-	  #verstring="-compatibility_version $minor_current -current_version $minor_current.$revision"
-	  # make the compatibility version match the Xcode project files, i.e. 1.0
-	  xlcverstring="${wl}-compatibility_version 1.0 ${wl}-current_version ${wl}$minor_current.$revision"
-	  verstring="-compatibility_version 1.0 -current_version $minor_current.$revision"
+	  xlcverstring="${wl}-compatibility_version ${wl}$minor_current ${wl}-current_version ${wl}$minor_current.$revision"
+	  verstring="-compatibility_version $minor_current -current_version $minor_current.$revision"
 	  ;;
 
 	freebsd-aout)

